iPSL/Electrical/Loads/PSSE/BaseClasses/baseLoad.mo

```diff
@@ -22,15 +27,15 @@ partial model baseLoad
   Real Q "Reactive power consumption (pu)";
 protected
   parameter Real anglev_rad=angle_0*pi/180;
-  parameter Real p0=S_i.re*V_0 + S_y.re*V_0^2 + S_p.re;
-  parameter Real q0=S_i.im*V_0 + S_y.im*V_0^2 + S_p.im;
+  parameter Real p0=(S_i.re*V_0 + S_y.re*V_0^2 + S_p.re)/S_b "pu";
+  parameter Real q0=(S_i.im*V_0 + S_y.im*V_0^2 + S_p.im)/S_b "pu";
   parameter Real vr0=V_0*cos(anglev_rad) "Initialitation";
   parameter Real vi0=V_0*sin(anglev_rad) "Initialitation";
   parameter Real ir0=(p0*vr0 + q0*vi0)/(vr0^2 + vi0^2) "Initialitation";
   parameter Real ii0=(p0*vi0 - q0*vr0)/(vr0^2 + vi0^2) "Initialitation";
-  parameter Complex S_P=Complex((1 - a.re - b.re)*S_p.re, (1 - a.im - b.im)*S_p.im);
-  parameter Complex S_I=S_i + Complex(a.re*S_p.re/V_0, a.im*S_p.im/V_0);
-  parameter Complex S_Y=S_y + Complex(b.re*S_p.re/V_0^2, b.im*S_p.im/V_0^2);
+  parameter Complex S_P=(Complex((1 - a.re - b.re)*S_p.re, (1 - a.im - b.im)*S_p.im))/S_b "pu";
```

Comment:
FYI: Unfortunately there doesn't exist (yet: [Modelica#911](https://trac.modelica.org/Modelica/ticket/1911)) a complex variant of the per-unit quantity.